cmd/zetaclientd/start.go

```diff
@@ -333,9 +333,9 @@ func promptPasswords() (string, string, error) {
 		return "", "", err
 	}
 
-	if TSSKeyPass == "" {
-		return "", "", errors.New("tss password is required to start zetaclient")
-	}
+	//trim delimiters
+	hotKeyPass = strings.TrimSuffix(hotKeyPass, "\n")
```

Comment:
these two lines are to allow empty pass for backwards compatibility? just to confirm

Reply:
Yes, the read function includes the line feed character so we need to remove them